terraform/provisioning/flatcar.bash

```diff
@@ -21,7 +21,84 @@ EOF
   sudo systemctl restart docker
 }
 
+install_docker_userns() {
+# add a new user for this specifically
+sudo useradd dockerns
+
+cat <<EOF | sudo tee /etc/subuid
+dockerns:231072:65536
+EOF
+cat <<EOF | sudo tee /etc/subgid
+dockerns:231072:65536
+EOF
```

Comment:
It should also work with
```
  sudo touch /etc/subuid
  sudo touch /etc/subgid
```

And keeping `--userns-remap=default`. I see that the docker daemon automatically populated the files with:
```
core@ip-10-0-0-124 ~ $ cat /etc/subuid
dockremap:100000:65536
core@ip-10-0-0-124 ~ $ cat /etc/subgid
dockremap:100000:65536
```

Maybe better to stay close to the default setup?

Reply:
done